tests/test_view_model.py

```diff
@@ -0,0 +1,25 @@
+"""Unit tests for view_model.py"""
+import pytest
+from todo_app.view_model import ViewModel
+from todo_app.item import Item
+
+def test_todo_items():
+    items = [
+                Item(1, "Test_ToDo1", "To Do"),
+            ]
+    todo_items_list = ViewModel(items).todo_items
+    assert len(todo_items_list) > 0, "No ToDo item found"
```

Comment:
Your tests aren't really testing that the filtering works - they would all pass if the `todo_items` etc properties simply returned `self._items`.

You should put more than one item into the view model.

Reply:
Can include these comments in my Module 4 PR